packages/file_selector/file_selector/lib/file_selector.dart

```diff
@@ -21,7 +33,17 @@ Future<XFile?> openFile({
       confirmButtonText: confirmButtonText);
 }
 
-/// Open file dialog for loading files and return a list of file paths
+/// Open file dialog for loading files and return a list of file paths.
+///
+/// [acceptedTypeGroups] is the file type that can be selected in the dialog.
+/// there are differences in behavior depending on the platform as follows.
+/// - Windows: Each group will be an entry in a list of filter options.
+/// - macOS: The union of all types allowed by all of the groups will be allowed.
+///
+/// [initialDirectory] is the full path to the directory that will be displayed when the dialog is opened.
+///
+/// [confirmButtonText] is the text in the confirmation button of the dialog.
+/// when omitted, the default OS label. (for example, "Open")
 Future<List<XFile>> openFiles({
```

Comment:
It should be documented that this can return an empty list, such as when the user cancels.